src/common/state/DataNode.C

```diff
@@ -511,6 +511,14 @@ DataNode::FreeData()
             delete mptr;
         }
         break;
+    case MAX_NODE_TYPES: // unused
```

Comment:
Why not use `default` label instead? It would be a tad more future-proofed that way.

Reply:
thanks, that solves the issue as well, I will update to use that strategy